### ISSUE TITLE
SC 247 -- Visible focus indicator missing on accordion

### ISSUE THREAD

**Issue by damiansian** [newjersey/accessibility-pm#9]
## URL 
https://demo.fileyourstatetaxes.org/en/nj/landing-page

## Issue
The accordions/disclosure components have no visible focus indicators for keyboard focus. 

## WCAG
[SC 2.4.7: Focus Visible (Level AA)](https://w3.org/TR/WCAG21#focus-visible):

## User impact
Sighted users with physical disabilities that rely on keyboards to operate digital interfaces need a visible focus indicator to understand what control has focus so it can be operated.  

### Should be
```css
.xxx:focus {
    box-shadow: 0 0 0 5px #21C834,inset 0px 2px 0px rgba(0,0,0,0.15)
}
```

**Comment by amanifarooque** [comment]
Hi @damiansian! Are these two separate issues (what's indicated in the title vs. the body of this item)? 
- no focus on the accordion
- focus state lacks sufficient contrast?

**Comment by damiansian** [comment]
@amanifarooque 

Sorry, this one (Issue 9) is saying the disclosure component has no visible focus. The next one (Issue 10) says the authored focus style for other interactive controls does not meet contrast minimums.

**Comment by amurthy2** [comment]
@mmazanec22 looks like USWDS has the focus state around the box and it's triggered on click as well as tab. could we incorporate those updates? https://designsystem.digital.gov/components/accordion/

**Comment by damiansian** [comment]
Tested on this URL: 
https://demo.fileyourstatetaxes.org/en/nj/landing-page

<img width="592" alt="Image" src="https://github.com/user-attachments/assets/7a63cea7-5d7d-4a9e-a966-786d3a4d29e0" />

Exceeds minimum contrast of 3:1. The offset on the focus ring allows us to measure the contrast on #FFFFFF instead of the #52DAF2 of the disclosure component.
